pyproject.toml

```diff
@@ -68,13 +68,14 @@ uvloop = {version = "^0.19.0", markers = "sys_platform != 'win32'"}
 hiredis = {version = "^2.2.3", optional = true}
 redis = {version = "^5.0.1", optional = true}
 docker = {version = "^6.1.3", optional = true}
+async-timeout = "^4.0.3"
```

Comment:
```suggestion
async-timeout = {version = "^4.0.3", optional = true}
```

Reply:
Thanks @benjaminsingleton I can make this change once it lands on `develop`.